Java 21 is not supported with the gradle version libgdx setup uses

I haven't found an issue in github about this so here's one:

#### Issue details
Setup app uses gradle 7.5.1 which is outdated and doesn't support java 21.

See here for gradles compatibility matrix: https://docs.gradle.org/current/userguide/compatibility.html

#### Reproduction steps/code
Install a java 21 JDK, try and generate a new project with the setup app, it'll fail with `Unsupported class file major version 63`

#### Version of libGDX and/or relevant dependencies
1.12.1

#### Stacktrace
```java
Generating app in C:\Users\xxx\Downloads\test
Executing 'C:\Users\xxx\Downloads\test/gradlew.bat clean --no-daemon'
To honour the JVM settings for this build a single-use Daemon process will be forked. See https://docs.gradle.org/7.5.1/userguide/gradle_daemon.html#sec:disabling_the_daemon.
Daemon will be stopped at the end of the build 

FAILURE: Build failed with an exception.

What went wrong:
Could not open settings generic class cache for settings file 'C:\Users\xxx\Downloads\test\settings.gradle' (C:\Users\xxx.gradle\caches\7.5.1\scripts\13dbo4zft16dyvv6xki9uzovd).
BUG! exception in phase 'semantic analysis' in source unit 'BuildScript' Unsupported class file major version 63

Try:
Run with --stacktrace option to get the stack trace.
Run with --info or --debug option to get more log output.
Run with --scan to get full insights.

Get more help at https://help.gradle.org/

BUILD FAILED in 6s
Done!
To import in Eclipse: File -> Import -> Gradle -> Existing Gradle Project
To import to Intellij IDEA: File -> Open -> build.gradle
To import to NetBeans: File -> Open Project...
```

#### Please select the affected platforms
- [ ] Android
- [ ] iOS
- [ ] HTML/GWT
- [X] Windows
- [ ] Linux
- [ ] macOS

--- a/extensions/gdx-setup/res/com/badlogic/gdx/setup/resources/gradle/wrapper/gradle-wrapper.properties
+++ b/extensions/gdx-setup/res/com/badlogic/gdx/setup/resources/gradle/wrapper/gradle-wrapper.properties
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.4-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.5-bin.zip
 networkTimeout=10000
 validateDistributionUrl=true
 zipStoreBase=GRADLE_USER_HOME

--- a/extensions/gdx-setup/src/com/badlogic/gdx/setup/Language.java
+++ b/extensions/gdx-setup/src/com/badlogic/gdx/setup/Language.java
@@ -4,7 +4,7 @@ package com.badlogic.gdx.setup;
 public enum Language {
 	JAVA("java", "", "", "",
 		"java-library;java-library;java-library;com.android.application;java-library,robovm;java-library,gwt,war,org.gretty",
-		true), KOTLIN("kotlin", "ext.kotlinVersion = '1.3.41'",
+		true), KOTLIN("kotlin", "ext.kotlinVersion = '1.9.21'",
 			"classpath \"org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion\"",
 			"api \"org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion\"",
 			"kotlin;kotlin;kotlin;android,kotlin-android;kotlin,robovm; ", false);

